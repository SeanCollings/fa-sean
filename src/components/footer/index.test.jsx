describe('footer unit test', () => {
  test('should render', () => {});
});
