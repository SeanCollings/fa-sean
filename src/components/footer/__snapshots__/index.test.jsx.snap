// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`footer unit test should render 1`] = `
<footer
  className="sc-bdfBwQ fvPrKx"
>
  <div
    className="sc-gsTCUz bFsUWP"
  >
    <div>
      <a
        className="sc-dlfnbm cDFxQC"
        href="https://www.theiconic.com.au/"
        target="_blank"
      >
        The Iconic
      </a>
    </div>
  </div>
</footer>
`;
